pkg/reconciler/v1alpha1/route/table_test.go

```diff
@@ -1589,6 +1622,15 @@ func patchAddLabel(namespace, name, key, value, version string) clientgotesting.
 	return action
 }
 
+func patchLastPinned(namespace, name string) clientgotesting.PatchActionImpl {
+	action := clientgotesting.PatchActionImpl{}
+	action.Name = name
+	action.Namespace = namespace
+	patch := `{"metadata":{"annotations":{"serving.knative.dev/lastPinned":"1000000000"}}}`
```

Comment:
If this constant needs coordination with another can you make this a fmt string?